### ISSUE TITLE
Docs fix: Flags should come before BASEDOMAIN

### ISSUE THREAD

**Issue by vparekh94** [astronomer/astro-cli#149]
When attempting to use a flag:
```
astro auth login astronomer.cloud -o
```

Errors:
```
Error: accepts at most 1 arg(s), received 2
Usage:
  astro auth login [BASEDOMAIN] [flags]

Flags:
  -h, --help    help for login
  -o, --oauth   do not prompt for local auth
```

However, 
```
astro auth login -o astronomer.cloud

 CLUSTER                             WORKSPACE                           
 astronomer.cloud                    ab4ce7f7-6dd7-4b1f-837d-0fbb43e4924f

 Switched cluster

Please visit the following URL, authenticate and paste token in next prompt

https://app.astronomer.cloud/login?source=cli

oAuth Token: 
```

works perfectly. The help/error message above should indicate that the `-o` flag comes _before_ the BASEDOMAIN.

**Comment by benjamingregory** [comment]
@vparekh94 

I just tried `astro auth login astronomer.cloud -o` on v0.7.1 and it worked as expected. astro auth login -o astronomer.cloud` is also valid.

Can you confirm you're still having this issue on the most recent CLI release?

**Comment by vparekh94** [comment]
@benjamingregory I believe this was for a 0.6.2 issue - looks like it's resolved with 0.7.1.
